src/PrestaShopBundle/Entity/Translation.php

```diff
@@ -60,7 +60,7 @@ class Translation
     /**
      * @var string
      *
-     * @ORM\Column(name="`key`", type="text", length=65500)
+     * @ORM\Column(name="`key`", type="binary", length=65500)
```

Comment:
```suggestion
     * @ORM\Column(name="`key`", type="varbinary", length=65000)
```

Because of this:
> #1118 - Row size too large. The maximum row size for the used table type, not counting BLOBs, is 65535. This includes storage overhead, check the manual. You have to change some columns to TEXT or BLOBs

Reply:
I don't see varbinary in the list of supported type in doctrine? but yes the length is a bit large for a key anyway, we can put type="binary", length=8000 to be safe for example?

Reply:
Good idea. Not sure we will have a key greater than 8000 😅

Reply:
I propose this key: realarmbrotherinterestguessfieldpoorlowideachargemilkstoodnamesightmotionthereeightsitcanabovewarfearonclasswomenmarkethitcarchicktopranfruitlowonbeforeskineventbookdifferscorelatetowhylocateinstrumentmeetgoodcaughtrangedatawhitetellcloudroomlotsquarekeptholedogwinpleaseblowropeofmultiplysciencestargojusttellhorseshinerockcaregreenspringthingscoreloghuntbeatflowermajorbornplaymodernburnclotherockmarkdrivenextbrokeoccurthirdpreparedressbehindtiegrewleastrestbitprocess24reasonskillmarketpowersquareworksurpriseneighborindustrytowardfootwarmdoublehimmasterremoteroadwestproductherlessrollplanetforwardcoverwin90ispullmixneedsmellputfreerichwontthroughoverwellspiderlawtosubstancewheresaladholeviewinchsentencespiderserveproducemuchtailblowmymasterstreamresultplainmuchskilldearhomemarketfillcloudconnectmagnetwhileallowwhitemarketwarcenturyswimartheremadealsobeatnaturaldoghappenaddsettlesonshellvowelstudyroundwarmamonguntilnearclockbonebirdhaslovecoursemethodbothgasdoessystemmilkwearonceswimboughtsummershortvarystandbluewhatcenterloudsentencesomethiscurrentdrawathardthuswheelloudnecessaryprintcenterfeetlikethuspatternhischargegaveexperimentrightfamily]totaledgeheremonthgirlstepapachepostmultiplyspacecropengineexpectsupportlatenearrunaresoldiergoodselllowanythreedifferfamilytravelprobablecostleadchildrensaidcamelikehitshowstarremoterememberhotdeterminetradelengthfivetellhesimilardownfrontbitfairsymbolseedmillionnosemeasuredecimalrightcarposecenturysubtracttieteachmountainorganeffectbellforwardthosenotepluralwomenknowsunsupportladyinsectconsidermixserveoppositedetroitrubbigdoesnocarepickwarmboxthickpaythinkfinehurrystudydownloaddegreedrinkwestspotwrittensuffixhelpenginecaughtsubjectspeechhugehappenfirstfinalspeakdependletskillkeepmigrateareapartylandfindfruitfindperiodrealexperiencestandmypickgiverulesouthwearscalebatsmellpatternthankprovideselectcoldnothingsavedistantseveralshortstepfillwefingerairaddtakesatsofttrianglerailsimplegardenofficetrafficquestionunitlittletermflowercertainteamordersharphisdecimalbutfallgarageothersingupamongstreamsideageoptimumcourselegbellagainhearsuitallowlakeahotdesignwatchminecleanwallagowindowsliphelpserveeventeverspotlovesideperiodquickpitchmysideshouldersamelegdressnoticegreensteptravelreadycampprocesslinuxbreakmapwinwentmajorcityplacegrassreadheardmillionwearlearnchiefdeathspecialsitmultiplydownloadalsonaturalmoonrulecomeredcertainsondeepfewsegmentsummernoticecloseeitherliftblueyourmiletopnorthtrycaptainnaturechangespokeclassoutartsinglestaticraincase58goldmissweredryanytreeduringbringbatwoodyearperhapsgladmeetpoemmountainmeantcomedivisionfriendso89branchspendtogethersecondspiderusualsmallsamemanypleaseservelakepleaserailperioddearsuffixfriendsheetmonthdogeventhearpeoplecreateclaimorsystemprobablemountainbeautyunitsteamenergybreadstoodwrotelawshethatsailpresentfullstraightusualsharpinterestresultroadheartregionreadfoundsteadpowerbasehurrysentenceexercisesteeldecideskilllegthousandspendweresailmayyoungwordscorelengththishadtoastguidethesefathernorthsupplysaychairmadeforcequartusehurryledlotmixsomethreebandgame46seaseatchargeparagraphafraidfiveherewellasreplymadeshellsurfacechanceallsupportbodyinventmainwholeftfromsolutionpassmultiplyrowloudnounstoodknewfollowcamppagetiejumpsubjectspeakearspellsoonaboveneedfronthugewenttrackdancestaythingtreeslowsawdisksetteethserverwroteprocesslinuxhighbaconbyarriveasdanceliquidatdoneclaimfatgamedollarbottomsmallwerefamilytrafficgivetowardlaughcomerestdiebloodroadminutemanylonemapmouseexciteeventhurrymiddleturnraidshortsincecaughttimecenturymetermpeoplecorngathermomentdetroitinventgroupspecialcleansonpopulatesolvehairstickblackboardexercisemineleftmattercampwallracehisbottomorbadrepeathappenloudplansavedryteethchiefpastmorninghorsehighpeoplebellinstantsleepcapitalfaceholdwellcoolsentdifficultbeatwithsayhumanletalsolettypeholddependboatofferyoursooncellspecialplantshellhourwouldfarmeroseexperimentwifelongincludelotengineleadbrighttoastcenturyapachehugehistorystationplaymaterialjoymuststretchcaptainseemgreatgaragepossibleshineregionsurpriseteamlesswindowreplytailmeanconnectstatefeetwerefishpopulatelookfollowworldsaltlifewaysmileseatnounwhenmomentfigmarkpositionthankspreadlongroundparentpreparetopanshaperightsuccesssimilarweightsuffixexamplelovesavewatchtripmaincausesoillettercommonmysecondplayshoecourseabovespeaksecondnumeralhearttraincurrentsandroomremotebellbedparagraphsmellhatwaitrememberwouldbornwatchgamesecondtrucksentsawstaysightthenpelicanservethreestudenttradeideastraightmiddlegasdoessuffixgoneveryendresultmightlightturnmoneyhugewithkeepohletparenthumanwindhuntcellmomentspeechnationabovethingenergysavefriendblockhebarswimminuteshoulderringtoastorganliftfactlanguagewesteveningmerubbyteachsavefrienddiecapitaltubewaterbroketimeevenshoredropeveningeveningoptimumnumbertroubledontwhilegreen 😅

Reply:
Is this a valid API token ? :trollface: